qiskit/providers/ibmq/managed/managedjobset.py

```diff
@@ -187,20 +196,20 @@ def retrieve_jobs(self, provider: AccountProvider, refresh: bool = False) -> Non
             experiment_index = mjob.end_index + 1
 
     def statuses(self) -> List[Union[JobStatus, None]]:
-        """Return the status of each job.
+        """Return the status of each job in this set.
 
         Returns:
             A list of job statuses. The entry is ``None`` if the job status
-                cannot be retrieved due to server error.
+            cannot be retrieved due to server error.
```

Comment:
It might be possible to rewrite this as: `A list of job statuses or ``None`` if the job statuses cannot be retrieved due to a server error`?

Reply:
Same as above.